### PR TITLE
Add `-g` automatically when building an interpreter with yk debug mode.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: ykcapi/scripts/yk-config [hunk 1]
```diff
@@ -30,10 +30,16 @@ handle_arg() {
...
             OUTPUT="${OUTPUT} -fyk-noinline-funcs-with-loops"
+            case $mode in
+                debug) OUTPUT="$OUTPUT -g" ;;
+            esac
             ;;
         --cppflags)
             # Path to yk.h
             OUTPUT="${OUTPUT} -I${DIR}/.."
+            case $mode in
+                debug) OUTPUT="$OUTPUT -g" ;;
```
[the remark this answers is not in the document]
Comment (ltratt) [reply ykcapi/scripts/yk-config]: Oops, I misread "cpp" as "C++"! Fixed in bb1ce70.

**Comment by vext01** [comment]
Please squash!

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**